### PR TITLE
Minimize agw container build context

### PR DESCRIPTION
**Pull request by crasu**
## Summary

* Reduce context size for agw python container

## Test Plan

* LTE attach integ test works

### CODE REVIEW

File: lte/gateway/docker/services/python/Dockerfile [hunk 1]
```diff
@@ -60,7 +60,10 @@ RUN apt-get update && apt-get install -y \
 
 RUN gem install fpm
 
-COPY . $MAGMA_ROOT/
+#COPY . $MAGMA_ROOT/
```
Comment (crasu) [inline lte/gateway/docker/services/python/Dockerfile]: remove commented out code

File: lte/gateway/docker/services/python/Dockerfile [hunk 2]
```diff
@@ -121,28 +146,30 @@ RUN echo "deb https://packages.fluentbit.io/ubuntu/focal focal main" > /etc/apt/
...
+COPY ./orc8r/gateway/configs/templates/nghttpx.conf.template /etc/magma/templates/nghttpx.conf.template
+COPY ./lte/gateway/deploy/roles/magma/files/set_irq_affinity /usr/local/bin/set_irq_affinity
+
 COPY --from=builder /build /build
-COPY --from=builder /magma /magma
-COPY --from=builder /magma/orc8r/gateway/python/scripts/ /usr/local/bin
-COPY --from=builder /magma/lte/gateway/python/scripts/ /usr/local/bin
+COPY --from=builder /magma/lte/gateway/python/magma /magma/lte/gateway/python/magma
+COPY --from=builder /magma/orc8r/gateway/python/magma /magma/orc8r/gateway/python/magma
 COPY --from=builder /var/tmp/python3-aioeventlet* /var/tmp/
```
Comment (crasu) [inline lte/gateway/docker/services/python/Dockerfile]: we probably want to make it clearer that we copy over some debs here.

Ike copying /var/tmp/python3-aioeventlet*.deb

File: lte/gateway/docker/services/python/Dockerfile [hunk 3]
```diff
@@ -121,28 +146,30 @@ RUN echo "deb https://packages.fluentbit.io/ubuntu/focal focal main" > /etc/apt/
...
-COPY --from=builder /magma/orc8r/gateway/python/scripts/generate_nghttpx_config.py /usr/local/bin/generate_nghttpx_config.py
-COPY --from=builder /magma/orc8r/gateway/python/scripts/generate_service_config.py /usr/local/bin/generate_service_config.py
-COPY --from=builder /magma/orc8r/gateway/python/scripts/generate_fluent_bit_config.py /usr/local/bin/generate_fluent_bit_config.py
-COPY --from=builder /magma/lte/gateway/deploy/roles/magma/files/set_irq_affinity /usr/local/bin/set_irq_affinity
 
 RUN chmod -R +x /usr/local/bin/generate* /usr/local/bin/set_irq_affinity /usr/local/bin/checkin_cli.py && \
   dpkg -i /var/tmp/python3-aioeventlet* && \
   pip install jsonpointer>$JSONPOINTER_VERSION && \
   mkdir -p /var/opt/magma/
+
```
Comment (crasu) [inline lte/gateway/docker/services/python/Dockerfile]: remove extra lines

File: lte/gateway/docker/services/python/Dockerfile [hunk 4]
```diff
@@ -69,6 +72,28 @@ RUN mkdir -p ${SWAGGER_CODEGEN_DIR}; \
...
 WORKDIR /magma/lte/gateway/python
+COPY \
+    ./lte/gateway/python/Makefile \
+    ./lte/gateway/python/defs.mk \
+    ./lte/gateway/python/setup.py \
+    $MAGMA_ROOT/lte/gateway/python/
+COPY ./lte/gateway/python/load_tests $MAGMA_ROOT/lte/gateway/python/load_tests
+COPY ./lte/gateway/python/scripts $MAGMA_ROOT/lte/gateway/python/scripts
+COPY ./orc8r/gateway/python/magma $MAGMA_ROOT/orc8r/gateway/python/magma
+COPY \
```
Comment (crasu) [inline lte/gateway/docker/services/python/Dockerfile]: you probably want to include the entire orc8r/gateway/python part

Comment (wolfseb) [reply lte/gateway/docker/services/python/Dockerfile]: In my testing it was enough to just include the service code base, the makefiles, and the scripts. Looking through it again, that is pretty much the entire `orc8r/gateway/python` folder, so for simplicity that is probably better, yes.

Comment (wolfseb) [reply lte/gateway/docker/services/python/Dockerfile:83]: We could do the same for the `lte/gateway/python` folder, but we do not need the `integ_test` folder here (we could delete it after copying, which would make the Dockerfile at least more readable)

**Pull request closed without merging**